server/api/lib/tags/utils.js

```diff
@@ -1,39 +1,67 @@
 const PlanAreaChanges = require('../../../api/model/plan_area_changes');
-const { tagDataRules } = require('../../constants');
+const { tagDataRules, area_change_types } = require('../../constants');
 
 const isTagByUsageAddition = async (planId, rule) => {
 	try {
 		const result = await PlanAreaChanges.byPlanAndUsage(planId, rule.usage);
 		if (result && result.models && result.models[0]) {
 			const changeToApprovedState = result.models[0].attributes.change_to_approved_state;
-			if ( (changeToApprovedState.length > 1) && 
-				(changeToApprovedState.substring(0,1) === '+') ) {
-				const change = Number(changeToApprovedState.replace('+','').replace(',',''));
-				if (change >= Number(rule.minValue)) {
-					return {   
-						created_by_data_rules: `{rule:'${rule.description}',detail:'adds ${changeToApprovedState} ${rule.usage}'}`
-					}; 
-				}
-			}
-
+			const approvedState = result.models[0].attributes.approved_state;
+			switch (rule.changeType) {
+				case area_change_types.INCREASED_USAGE:
+					// change to approved state
+					if ((changeToApprovedState.length > 1) && 
+						(changeToApprovedState.substring(0,1) === '+') ) {
```

Comment:
Can this be documented in a comment? What the replace of `+` and `const change = Number(changeToApprovedState.replace('+','').replace(',',''));` are for?

Reply:
please take a look on an land change service I'm adding on this PR: https://github.com/meirim-org/meirim/pull/425/files

We can think of integrating it to the tag functionality later on